src/app/initialisation/syncPoolService.js

```diff
@@ -43,14 +48,22 @@ function SyncPoolService($q, mediator, workorderSync, workflowSync, resultSync,
     promises.push(workflowSync.createManager());
     promises.push(messageSync.createManager());
     promises.push(resultSync.createManager());
-    return $q.all(promises).then(function(managers) {
-      var map = {};
-      managers.forEach(function(managerWrapper) {
-        map[managerWrapper.manager.datasetId] = managerWrapper;
+
+    //Initialising the sync managers for the required datasets.
+    return syncService.manage(config.datasetIds.workorders, {}, {}, config.syncOptions)
```

Comment:
Actually can we just pull that to modules itself as suggested in other PR's with some default sync config for each datasets. Users would be able to override configs if they needed etc.